src/ingredients.ts

```diff
@@ -1,123 +1,82 @@
-        .then(res => res.json())
+export const createIngredientsListFromDb = async () => {
+    const receivedIngredients = await getDataFromDb(ingredientsPath)
 
     createListItems(receivedIngredients)
     ingredientList = ingredientList.concat(receivedIngredients)
 
-    console.log(receivedIngredients)
     return receivedIngredients
 }
 
 export const updateIngredientsList = () => {
-    ingredientsSet.delete(undefined)
-    ingredientsSet.delete(null)
-    Storage.set('ingredients', Array.from(ingredientsSet))
     $('.content__list').remove()
     $('<div>').addClass('content__list').appendTo('.content')
 }
 
-export const createIngredients = async () => {
+export const createNewIngredient = async () => {
     const name = $('.content__input').val() as string
-    console.log(name)
-    const insertNewIngredient = await fetch('http://localhost:3001/ingredient', {
-        method: 'POST',
-        headers: {
-            'Content-Type': 'application/json;charset=utf-8',
-            'Access-Control-Allow-Origin': '*'
-        },
-        body: JSON.stringify({ name })
-    })
+    const newIngredient = await insertNewElement({ name }, ingredientsPath)
     
-    console.log(insertNewIngredient)
-    return insertNewIngredient
-    // ingredientsSet.add(newValue)
-    // updateIngredientsList()
-
-    // ingredientsSet.forEach(text => {
-    //     const $newListItem = $('<div>').addClass('content__list-item')
-    //     const $spanBtns = createSpanBtns()
-
-    //     $newListItem
-    //         .text(text)
-    //         .append($spanBtns)
-    //         .appendTo($('.content__list'))
-    // })
+    updateIngredientsList()
+    createIngredientsListFromDb()
+
+    return newIngredient
 }
 
 const createListItems = ingredientList => {
-    ingredientList.forEach(text => {
+    ingredientList.forEach(ingredient => {
         const $newListItem = $('<div>').addClass('content__list-item')
         const $spanBtns = createSpanBtns()
 
         $newListItem
-            .text(text.name)
+            .text(ingredient.name)
+            .attr('id', `${ingredient.ingredientId}`)
             .append($spanBtns)
             .appendTo($('.content__list'))
     })
 }
 
-const removeIngredient = (event: JQuery.ClickEvent) => {
-    const selectedItem: string = $(event.target).closest('div').text()
-    const id = ingredientList.reduce((acc, ingredient) => {
-        const ingredientIdIndex = 0
-        const nameIndex = 1
-
-        if (Object.values(ingredient).at(nameIndex) === selectedItem) {
-            acc = Object.values(ingredient).at(ingredientIdIndex)
-        }
-        return acc
-    }, 0)
-
-    console.log(id)
-    // console.log(Object.values(removeIngredientId).at(ingredientIdIndex))
-    
-    const deleteIngredient = fetch('http://localhost:3001/ingredient', {
-        method: 'DELETE',
+const patchIngredient = async (id, name: string) => {
+    const editedIngredient = await axios({
```

Comment:
no await promise if you're not doing anything with it. Simply return the promise